_posts/2018-11-02-cpp-array.md

```diff
@@ -0,0 +1,153 @@
+---
+title: C++ persistent containers - array
+author: igchor
+layout: post
+identifier: cpp_array
+---
+
+### Introduction
+
+Until now, our C++ bindings were missing one important component - persistent
+containers. In 1.5 release we have introduced the first one - [pmem::obj::array][cpp_array].
+This container is currently placed in `experimental` namespace and folder. It has
+almost the same functionality as `std::array` from C++11 but takes care of adding
+elements to a transaction and guarantees a fixed layout. API for `pmem::obj::array`
```

Comment:
elaborate on the fixed layout. Maybe change it to 'stable in-memory layout'/`stable ABI` ?